### PR TITLE
Fix build.ts type error

### PR DESCRIPTION
**Pull request by LukeSheard**
Fixes the issue with `any` assignment in `build.ts`

### CODE REVIEW

File: packages/modular-scripts/src/build.ts [hunk 1]
```diff
@@ -148,6 +148,8 @@ const typescriptConfig: TSConfig = {};
     throw new Error('Failed to load Typescript configuration');
   }
 
+  const exclude: string[] = (configObject.exclude as string[]) || [];
```
Comment (threepointone) [inline packages/modular-scripts/src/build.ts:151]: Could you add a comment here why this is necessary?

**Review by threepointone — COMMENTED** [review]
Please add a patch changeset.

File: packages/modular-scripts/src/build.ts (hunk 1 above)
Comment (LukeSheard) [reply packages/modular-scripts/src/build.ts:153]: Done

**Comment by LukeSheard** [comment]
I don't think this needs a changeset since there's nothing to release here as part of this change.

**Review by threepointone — APPROVED** [review]

**Pull request merged**